### DISCUSSION TITLE
Creating rating page

### DISCUSSION THREAD

**Discussion by HARDIK1221** [Sketchware-Pro/Sketchware-Pro#389] [Q&A] [answered]
I have created a app which takes values( rating from user) to firebase database. Now my question is that how can I find the average of the values entered by different users  from firebase DB ?

**Comment by j-px** [comment, marked as answer]
Simply go through every rating and add the rating values together, then, when you're done, divide that by the total number of reviews. Now you have the mean.

**Reply by HARDIK1221** [reply]
Thank you my problem is now fixed
